src/client/app/App.tsx

```
@@ -30,40 +31,46 @@ const RaceWorld = lazy(async () => {
 });
 
 const GithubIcon = () => (
-    <svg viewBox="0 0 24 24" fill="currentColor" className="w-8 h-8">
+    <svg viewBox="0 0 24 24" fill="currentColor" className="w-6 h-6">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)